dango/types/src/lending_pool.rs

```diff
@@ -0,0 +1,75 @@
+use std::collections::BTreeSet;
+
+use grug::{Addr, Coins, Denom};
+
+/// The namespace that lending pool uses.
+pub const NAMESPACE: &str = "lending";
+
+#[grug::derive(Serde)]
+pub struct InstantiateMsg {
+    pub whitelisted_denoms: BTreeSet<Denom>,
+}
+
+#[grug::derive(Serde)]
+pub enum ExecuteMsg {
+    /// Whitelist a denom. Can only be invoked by the owner.
+    WhitelistDenom(Denom),
+
+    /// Delist a denom. Can only be invoked by the owner.
+    DelistDenom(Denom),
```

Comment:
Instead of whitelist/delist, what we will do is to have a `Param` for each denom. A denom can be deposited/borrowed if it has a `Param`, otherwise not.